timely_beliefs/beliefs/classes.py

```diff
@@ -788,26 +857,102 @@ def rolling_viewpoint_accuracy(
         >>> df.rolling_viewpoint_accuracy(belief_horizon_window=(timedelta(days=1), timedelta(days=2)))
         >>> # Select the accuracy of beliefs formed 10 days beforehand with respect to 1 day past knowledge time
         >>> df.rolling_viewpoint_accuracy(belief_horizon=timedelta(days=10), reference_belief_horizon=timedelta(days=-1))
-        >>> # Select the accuracy of beliefs formed 10 days beforehand with respect to the latest belief formed by source 3
-        >>> df.rolling_viewpoint_accuracy(belief_horizon=timedelta(days=10), reference_source=3)
+        >>> # Select the accuracy of beliefs formed 10 days beforehand with respect to the latest belief formed by the first source
+        >>> df.rolling_viewpoint_accuracy(belief_horizon=timedelta(days=10), reference_source=df.lineage.sources[0])
 
         :param belief_horizon: timedelta indicating the belief should be formed at least this duration before knowledge time
         :param belief_horizon_window: optional tuple specifying a horizon window (e.g. between 1 and 2 days before the event value could have been known)
         :param reference_belief_horizon: optional timedelta to indicate that the accuracy should be determined with respect to the latest belief at this duration past knowledge time
         :param reference_source: optional BeliefSource to indicate that the accuracy should be determined with respect to the beliefs held by the given source
-        :param keep_reference_observation: Set to True to return the reference observation used to calculate mape and wape as a DataFrame column
+        :param df_observation: precomputed reference beliefs
```

Comment:
I feel this can be documented a bit more.

1. First, the term "observation" is synonymous with "reference" somehow to me here, can we make the connection between these concepts more clear?
2. This is also a BeliefsDataFrame, right?
3. I see that if `df_observations` is not passed in, we take the latest beliefs (on sefl) as observation. Let's mention that.
4. And I also see that `reference_source` is only used if we did *not* pass in `df_observations`. Mantion that or apply it in either case.
5. Why don't we have this parameter in `fixed_viewpoint_accuracy`, as well?

Reply:
Thanks. I also started renaming `df` to `bdf` so that any difference between a regular pandas DataFrame and a timely-beliefs BeliefsDataFrame will read more clearly.